pkg/utils/json.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package utils
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"openpitrix.io/openpitrix/pkg/logger"
+)
+
+func Loads(src string, dest interface{}) error {
+	err := json.Unmarshal([]byte(src), &dest)
+	if err != nil {
+		logger.Errorf("Failed to load json: %v", err)
```

Comment:
it is just a little function, the caller will do it when some error occurred

Reply:
The caller no need to logger again, just use this func is enough. With this log, we know an error exist, sometimes the error is not that critical.